doc/tutorial.md

```diff
@@ -691,6 +691,64 @@ fn calculator5() {
 }
 ```
 
+<a id="calculator7"></a>
```

Comment:
I can't tell what `calculator6` was supposed to be...seems like this could just be 6...